polaris-react/locales/en.json

```diff
@@ -155,6 +155,9 @@
       "clear": "Clear",
       "clearLabel": "Clear {filterName}"
     },
+    "InContextLearning": {
+      "accessibilityLabel": "Close tutorial"
```

Comment:
I got this name from this doc: https://docs.google.com/document/d/10x1ynHfpZVMDGKEhqIMhXhIT3suC8rdex4Dh1gbAJ3g/edit

I wasn't sure if we wanted to have a prop of the tutorial name passed into the component - at this point it seems unnecessary for just one label 🤷‍♀️  But I can update if we feel that:
1) having the name in this label is important
2) we have a use for the name beyond this label